CHANGELOG.md

```diff
@@ -1,6 +1,49 @@
 Moto Changelog
 ==============
 
+4.0.10
+-----
+Docker Digest for 4.0.10: <autopopulateddigest>
```

Comment:
Shouldn't these 4.0.10 be 4.0.11 @bblommers ?

Reply:
Yes, it should! Good catch @daybarr, and thanks for letting me know